serde-oracle-jdbc-json/src/main/java/io/micronaut/serde/oracle/jdbc/json/serde/OracleJsonLocaleDateTimeSerde.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.serde.oracle.jdbc.json.serde;
+
+import io.micronaut.core.annotation.Order;
+import io.micronaut.serde.support.serdes.LocalDateTimeSerde;
+import io.micronaut.serde.util.NullableSerde;
+import jakarta.inject.Singleton;
+
+import java.time.LocalDateTime;
+
+/**
+ * Serde for {@link LocalDateTime} from Oracle JSON.
+ *
+ * @author radovanradic
+ * @since 2.0.0
+ */
+@Singleton
+@Order(-100)
+public class OracleJsonLocaleDateTimeSerde extends OracleJsonTemporalSerde<LocalDateTime> {
```

Comment:
Maybe these serdes could be internal classes because using new `OracleType` annotation does the job now?

Reply:
Perhaps, @dstepanov recommends we change the tests and make the `OracleType` annotation unnecessary

Reply:
Changed in https://github.com/micronaut-projects/micronaut-serialization/pull/398/commits/905d53561b485d82cfa42f59503e25eedf4bfc73